_gsocproposals/2024/proposal_PodioONNX.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Enabling efficient PODIO data model integration with ONNX for training and inference
+layout: gsoc_proposal
+project: Key4hep
+year: 2024
+organization: 
+ - CERN
+ - DESY
+ - UManitoba
+difficulty: medium
+duration: 350
+mentor_avail: May-October
+---
+
+## Description
+
+[PODIO](https://github.com/AIDASoft/podio) is a data model definition package that provides the necessary functionality to generate C++ and Python code from a high level definition of an event data model in yaml format. EDM4hep is an example of a data model generated with PODIO, which serves as the common event data model for many HEP communities, brought together under the umbrella of the [Key4hep](https://github.com/key4hep) project.
```

Comment:
```suggestion
[PODIO](https://github.com/AIDASoft/podio) is a data model definition package that provides the necessary functionality to generate C++ and Python code from a high level definition of an event data model in yaml format. [EDM4hep](https://github.com/key4hep/EDM4hep) is an example of a data model generated with PODIO, which serves as the common event data model for many HEP communities, brought together under the umbrella of the [Key4hep](https://github.com/key4hep) project.
```
Maybe we should add EDM4eic here as well? In the end, they will probably work on that for prototyping, because you already have a few concrete examples I suppoe?